android/build.gradle

```diff
@@ -30,6 +30,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.19.+'
+    compile 'com.facebook.react:react-native:0.32.+'
```

Comment:
Hmm. Looks like they no longer distribute react-native aars via jcenter. The latest version is [0.20.1](https://bintray.com/bintray/jcenter/com.facebook.react%3Areact-native). I'm wondering how you get it to compile?